### PR TITLE
Keep it snimple snupples

### PR DESCRIPTION
**Pull request by comradekingu**
Happy to provide a nb_NO translation if this is merged. :)

### CODE REVIEW

**Comment by comradekingu** [comment]
@RomaricMourgues
My motivation is to have something I can support, and actually fits the meritocracy of libre software.
Moreover I don't even want there to be passive support for one of the many attacks on it in projects I volunteer my time in.
Since the project is up for translation, I'd rather get some fixes in before everyone is subjected to a more difficult task, and then having to do it again at some point down the road.

While some things are accidentally similar between the two in isolated fashion, they are meaningless in the bigger picture of the original, and my addition would be too it got plastered on.

It isn't "harassment-free" when it is defined as anything that is _perceived_ as harassment.
Much like there isn't any humanity without sexuality or community.
I don't want professionalism over all else, because that is the ethos of business, meaning profit before community.
I am not putting community before all else, I am saying professionalism alone is not community, by definition.
There is no intent, like there is no mitigation of conflict in informed conversation between involved parties.
Instead there is an escalation of conflict, purposefully in required secrecy, which
is an invitation for anti-social people to both do harm, and seek these positions.
The positions are further removed from the meritocracy of the project, and the "code"
itself is more important than anyone involved.
`aligned to this Code of Conduct`
It is also all-encompassing`
within project spaces and in public spaces`
It is never accepted by contributors, but works on a tacit assumption instead of choice.
There is also the assumption that someone with bad intent or anti-social pathology would
somehow avoid doing wrong for fear of something as banal as disobeying a text-file.
Vulnerable people believe this, and it creates false security.
Rosy language or not, it has never done anything but single people out for immutable characteristics,
and that is the opposite of normalization. Plenty of fiascos though.

TL;DR It is authoritarian, and I am not. You can have it, but then I can't help you.

**Comment by RomaricMourgues** [comment]
We took your comment into account. Licensing is still a hot topic discussed internally and with the community. In the meanwhile I will close this PR.
Thank you for your proposal and your detailed comment 🙏

**Comment by comradekingu** [comment]
@RomaricMourgues This is not licensing. Anyone is free to fork the project and remove it.
It is not possible to put requirements like "political attacks" under "unacceptable behavior" in a libre software/open source license.

Whoever this "we" is, I was asked for an explanation, and then not giveen one.
In saying it was authoritarian, and I seem to stand corrected.. It certainly is not the contributors, as they never pledged allegiance to this. Since its adoption and rewriting here, it is outdated by two versions.
Curiously you rewrote some intentionally draconian features, which is good.

As a challenge I'll ask what concept or content of value is lost.

**Comment by comradekingu** [comment]
@RomaricMourgues
Came her to check if Linagora had one of these, and it seems it is of so low importance that nobody bothered to put one up.
Is my understanding correct that the file that is several versions outdated here also carries over to that project, without being communicated in the slightest there, on the Weblate platform, or on https://twake.app/en/how-to-contribute/?
I further assume it will just be slapped on some day, without asking any contributors before or after. Or?

I did however find this https://twake.app/en/terms-of-service/
> Can not directly or indirectly be considered to be a harmful, deviant, abusive, illicit, unlawful activity nor an activity prohibited by law, rules, or regulations, whether national or international;

So it is still whatever sounds good, between "we welcome contributions from anyone" for what are contributions, and then claiming that makes for "diversity" when it comes to community, but then requiring zero deviancy.

Between my proposal, and what existed before it, there is no _both_.

Meanwhile "Our mission is to invent and develop ethical and open-source Good Tech - Join Us!"
The person that made the authoritarian CoC you have also made an "ethical" software license, and it is
_specifically_ not libre/open source. It seems you don't care at any level, but that makes it hard for principled people to contribute. For no reason at all.
